contributing_to_docs/term_glossary.adoc

```diff
@@ -218,7 +245,6 @@ This is the command line interface of OpenShift v3, previously referred to as
 the client tools in OpenShift v2.
```

Comment:
Double-check the versions

Do we still want to discuss v2 usage? This might be a good time to deprecate v2 references.

Reply:
I don't think so, no. Removing the latter part.